scripts/test_ci_test_with_retry.py

```
@@ -51,6 +54,64 @@ def run_wrapper(
+                "example.com/p/pkg",
+                "--",
+                "bash",
+                "-c",
+                r'''
+echo "$$" >"${CI_RETRY_STATE:?}"
+python3 -c 'import os, signal, time; signal.signal(signal.SIGTERM, signal.SIG_IGN); open(os.environ["CI_RETRY_READY"], "w").close(); time.sleep(3600)' &
+wait
+''',
+                "fake-go",
+            ]
+            process = subprocess.Popen(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)